README.md

```diff
@@ -1,3 +1,484 @@
+
+```bash
+git clone https://source.foundries.io/factories/<factory>/containers.git
+git mv gst-multi-camera-example.disabled gst-multi-camera-example
+git commit -s -m “gst-multi-camera-example” && git push
+```
+
+This compose-app requires a display connected, and the camera mezzanine for it to function properly.
+
+### Deploy
+
+After you push, the FoundriesFactory will build a new target. Once it is built, you may deploy as described below.
+
+---
+
+## Fioctl
+
+Fioctl is a command-line tool to manage your FoundriesFactory. To deploy applications listed below, you must have enabled them first as described above. The polling time on devices by default is five minutes, once you have issued your deployment command, please wait up to five minutes to see the change reflected.
+
+### Examples:
+
+### View Targets:
+  ```bash
+  ./fioctl targets list -f <factory>
+  ```
+### View Devices:
+  ```bash
+  ./fioctl devices list -f <factory>
+  ```
+### Change Device Tag:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --tags <tag_name>
+  ```
+### Change Device Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps <app_name1>,<app_name2>,<app_name3>
+  ```
+### Deploy shellhttpd Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps shellhttpd
+  ```
+### Deploy gst-concurrent-videoplay-composition Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-concurrent-videoplay-composition
+  ```
+### Deploy gst-ai-classification Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-classification
+  ```
+### Deploy gst-ai-daisychain-detection-classification Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-daisychain-detection-classification
+  ```
+### Deploy gst-ai-monodepth Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-monodepth
+  ```
+### Deploy gst-ai-object-detection Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-object-detection
+  ```
+### Deploy gst-ai-parallel-inference Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-object-detection
+  ```
+### Deploy gst-ai-pose-detection Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-pose-detection
+  ```
+### Deploy gst-ai-segmentation Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-segmentation
+  ```
+### Deploy gst-multi-camera-example Application:
+  ```bash
+  ./fioctl devices -f <factory> config updates <device_name> --apps gst-multi-camera-example
+  ```
+### Enable/Disable Wireguard VPN:
+  ```bash
+  ./fioctl devices -f <factory> config wireguard <device_name> enable/disable
+  ```
+
```

Comment:
```suggestion
#### View Targets:

  ```bash
  ./fioctl targets list -f <factory>
  ```

#### View Devices:

  ```bash
  ./fioctl devices list -f <factory>
  ```
#### Change Device Tag:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --tags <tag_name>
  ```

#### Change Device Application:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps <app_name1>,<app_name2>,<app_name3>
  ```

#### Deploy `shellhttpd`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps shellhttpd
  ```

#### Deploy `gst-concurrent-videoplay-composition`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-concurrent-videoplay-composition
  ```

#### Deploy `gst-ai-classification`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-classification
  ```

#### Deploy `gst-ai-daisychain-detection-classification`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-daisychain-detection-classification
  ```

#### Deploy `gst-ai-monodepth`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-monodepth
  ```

#### Deploy `gst-ai-object-detection`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-object-detection
  ```

#### Deploy `gst-ai-parallel-inference`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-object-detection
  ```

#### Deploy `gst-ai-pose-detection`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-pose-detection
  ```

#### Deploy `gst-ai-segmentation`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-ai-segmentation
  ```

#### Deploy `gst-multi-camera-example`:

  ```bash
  ./fioctl devices -f <factory> config updates <device_name> --apps gst-multi-camera-example
  ```

#### Enable/Disable Wireguard VPN:

  ```bash
  ./fioctl devices -f <factory> config wireguard <device_name> enable/disable
  ```

```

Reply:
I am almost at the end of my usual workday, but if urgent I can either get it to you later today or this weekend. Otherwise I can do it first thing next week alongside anything else I catch.

Reply:
Sent via email!